satpy/composites/__init__.py

```diff
@@ -1236,3 +1236,81 @@ def __call__(self, datasets, optional_datasets=None, **attrs):
 
         return super(SelfSharpenedRGB, self).__call__(
             (red, green, blue), optional_datasets=(high_res,), **attrs)
+##
+class Airmasstr(GenericCompositor):
+
+    def __call__(self, projectables, *args, **kwargs):
+        """Make an tropical airmass RGB image composite.
+
+        +--------------------+--------------------+--------------------+
+        | Channels           | Temp               | Gamma              |
+        +====================+====================+====================+
+        | WV7.3 - WV6.2      |   0.6 to 26.2 K    | gamma 1            |
+        +--------------------+--------------------+--------------------+
+        | IR10.4 - IR9.7     | -26.2 to 27.4 K    | gamma 1            |
+        +--------------------+--------------------+--------------------+
+        | WV6.2              | 208.5 to 243.9 K   | gamma 1            |
+        +--------------------+--------------------+--------------------+
+        """
+        ch1 = sub_arrays(projectables[1], projectables[0])
+        ch2 = sub_arrays(projectables[3], projectables[2])
+        res = super(Airmasstr, self).__call__((ch1, ch2,
+                                             projectables[0]),
+                                            *args, **kwargs)
+        return res
+
+
+class volcanic_gas(GenericCompositor):
```

Comment:
Please change to `VolcanicGas` to match PEP8 python styling guidelines. I'm actually surprised stickler didn't complain about this.